.github/workflows/deploy_conda.yml

```diff
@@ -0,0 +1,33 @@
+name: Conda Deployment
+on:
+  push:
+    branches: [master]
+    tags: ['v*']
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install deps on macos
+      if: contains( matrix.os, 'macos')
+      run: brew install coreutils || true
+
+    - name: Setup Conda and adccore checkout
+      env:
+        OS_NAME: ${{ matrix.os }}
+      run: |
+        bash ci/setup_conda.sh
```

Comment:
Might make sense to actually inline the script here for clarity. What do you think?

Reply:
I think it's a bit too long for inline use...

Reply:
But now we have this ugly `ci` folder in the root with just one entry ...

Reply:
Fair enough 😄

Reply:
Thanks, I appreciate it :smile: As a bonus now you just need to copy one file for the adcc conda deployment later ;).

Reply:
Why is there no easy way to get a platform-dependent miniconda installer script. So super ugly in the GHA setup...